clang/clang.go

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clang
```

Comment:
I lacked creativity for the name.:) `c` for commons, and `lang` for language-related utilities.

Reply:
Lol, that's a tough one. Maybe `cgo` as in "common Go utilities" :trollface: I'll sleep on it to see if I come up with any alternative names.

Reply:
Actually we use the `c` prefix for `cchan` (chan is a reserved word), `ccontext` and `csync` (both are package names in the stdlib). So here we might as well just use `lang`, there's no such package in the stdlib anyway. Just a first thought, I'll still see if there's anything better 🙃

Reply:
`lang` is good too. :+1: I was thinking about something that's related to memory or pointers, but I think it's too specific and we'd end up with only that or maybe just a couple of functions there.

Reply:
What other methods we'd expect to have? I'd imagine not so many? If so, maybe I'd name it simply `ptr` to indicate that's a package that deals with pointers.

Reply:
It's hard to tell, but as far as I remember, we don't use many such functions in our codebase. Having a `ptr` package is nice because of what you mentioned, the call is nice too (`ptr.Get`), but it would be great if it wasn't just a single function in the package.:)